src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TextUi;
+import duke.task.TaskList;
+
+/**
+ * The ByeCommand class is used when an exit command is given.
+ */
+public class ByeCommand extends Command {
+
+    /**
+     * Constructs a bye Command, which sets the boolean value isExit to true.
+     */
+    public ByeCommand() {
+        super("");
+        this.isExit = true;
+    }
+
+    /**
+     * Displays an exit message on the given user interface.
+     * @param tasks             List of tasks
+     * @param ui                User interface
+     * @param storage           Hard disk storage
+     * @return                  String containing Duke's response
+     */
+    @Override
+    public String execute(TaskList tasks, TextUi ui, Storage storage) {
+        String textToDisplay = "Bye. Hope to see you again soon!";
```

Comment:
I think the textToDisplay can be abstracted to the UI class since it fits their class better.